src/Explorer/Panes/PanelComponent.less

```diff
@@ -110,7 +110,34 @@
 .deleteCollectionFeedback {
   margin-top: 12px;
 }
+.addRemoveIcon {
+  margin-left: 4px !important;
+}
+.addRemoveIconLabel {
+  margin-top: 28px;
+  margin-left: 4px !important;
+}
+.addNewParamStyle {
+  margin-top: 5px;
+  margin-left: 5px !important;
+  cursor: pointer;
+}
 
 .panelGroupSpacing > * {
   margin-bottom: @SmallSpace;
 }
+.panelAddIconLabel {
```

Comment:
If we're going to reuse some of these constants in other panes, consider moving them into or using the constants from `Constants.less`.